templates/sso74-https.json

```diff
@@ -5,19 +5,19 @@
         "annotations": {
             "iconClass" : "icon-sso",
             "tags" : "sso,keycloak,jboss,hidden",
-            "version": "7",
-            "openshift.io/display-name": "Red Hat Single Sign-On CD 7 Tech Preview (Ephemeral with passthrough TLS)",
+            "version": "7.4.0.GA",
+            "openshift.io/display-name": "Red Hat Single Sign-On 7.4 on OpenJDK (Ephemeral with passthrough TLS)",
             "openshift.io/provider-display-name": "Red Hat, Inc.",
-            "description": "An example application based on RH-SSO CD 7 Tech Preview image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.",
-            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat Single Sign-On CD 7 server based deployment, securing RH-SSO communication using passthrough TLS.",
+            "description": "An example application based on RH-SSO 7.4 OpenJDK image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat Single Sign-On 7.4 OpenJDK server based deployment, securing RH-SSO communication using passthrough TLS.",
             "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-single-sign-on/",
             "template.openshift.io/support-url": "https://access.redhat.com"
         },
-        "name": "sso-cd-https"
+        "name": "sso74-https"
     },
     "labels": {
-        "template": "sso-cd-https",
-        "rhsso": "7"
+        "template": "sso74-https",
+        "rhsso": "7.4.0.GA"
```

Comment:
`s/7.4.0.GA/v7.4.0.GA/g`

Reply:
@iankko What is the justification for this suggestion?

Reply:
For the very least the name of the tag is used in the errata instructions to update the templates to a specific version for the very least, e.g. [RHEA-2020:0446](https://access.redhat.com/errata/RHEA-2020:0446) reads:

```
...
oc replace -n openshift --force -f \
https://raw.githubusercontent.com/jboss-container-images/redhat-sso-7-openshift-image/v7.3.6.GA/templates/${resource}
...
```

This used to be the tag naming scheme, we used till now. It can be changed, if desired, but in that case we will need to update it on all places previously referencing the `v.*` form

Reply:
Not needed, confused with the tag notation.